tests/integration/stepfunctions/v2/test_stepfunctions_v2.py

```diff
@@ -404,7 +404,7 @@ def check_invocations():
             assert {"Hello": TEST_RESULT_VALUE_2} == result.get("handled")
 
         # assert that the lambda has been invoked by the SM execution
-        retry(check_invocations, sleep=1, retries=10)
+        retry(check_invocations, sleep=10, retries=1000000)
```

Comment:
```suggestion
        retry(check_invocations, sleep=10, retries=1000000)
```

that seems a bit excessive :sweat_smile: